### ISSUE TITLE
[ACCESSIBILITY] - Missing alt text on images causing screen reader to announce "Unlabeled image"

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#510]
Android 2.2.11
Description
When swiping through the app, the initial focus as displayed in the screen capture below is announced as an "Unlabeled image." I don't know what image this is referring to.

To Reproduce

Go to Athletics 
With VoiceOver on, begin swiping through the screen.
When the list of Upcoming events reaches the one with the image, "Unlabeled image" will be announced.

Expected behavior
There should be an announcement of a description of the image (Alt text).

Smartphone (please complete the following information):
Device: Google Pixel 4a/ iPhone XR
OS: Android 10/ iOS 14.4

Screenshots
See the screen capture below
![Image_needs_alt_text PNG](https://user-images.githubusercontent.com/82892873/125796626-b2dde770-cb1d-4581-92f0-15c5ff4a4ee6.png)
![Image_needs_alt_text2](https://user-images.githubusercontent.com/82892873/125796636-2ae1cec0-5076-433b-bec2-d23e90fb7bf4.png)

Additional comments:
 Second screenshot is of the second occurrence of "Unlabeled image" which is on the Event detail screen of the event with featured image.  This is a content error most likely by whomever made the test content I just do not have access to correct the Alt text and check.   Please add alt text to all images in app and I will do another audit. Thank you.

**Comment by acovher2** [comment]
Hello, 
It has been 1 month; are there any updates on this issue? 
Please provide any information you can. Thank you.

**Comment by acovher2** [comment]
This works for this image only. Should I submit future request for client to get backend Alt text entry field which is required for any photo upload?  This is really what needs done if client can upload custom images in their backend. Please let me know.  Thank you.

**Comment by TodorBachvarov** [comment]
This will be pronounced for the images for each sport event. They all will be treated as sport image and will pronounce "Sports image". For now there is no need of entry field change.

**Comment by acovher2** [comment]
The images on events are no longer being pronounced at all by screen reader. Images require alt text for assistive technologies to pronounce. Every image on every screen needs to have this, even icons representing things. 
[WCAG 1.1.1 ](https://www.w3.org/TR/WCAG21/#dfn-non-text-content)

Expected behavior:
Currently the twitter feed is doing this correctly, images there are being pronounced by screen reader. 

Actual behavior:
Events and other places images are located, images need alt text

**Comment by TodorBachvarov** [comment]
I've added some description for the Athletics images. Added in version 3.3.8. If something still needs to be fixed please create separate ticket with detailed description and screenshots.

**Comment by acovher2** [comment]
This will not just be the athletics images it will be any image anywhere in the app. Images on events, images on groups etc. I think we need an accessible image uploader and a custom alt text field required. I will close this ticket and open feature requests.
